pep-0649.rst

```diff
@@ -713,15 +728,18 @@ makes some modifications to the annotations before it returns them.
 This PEP adds a new keyword-only parameter to these two functions,
 ``format``.  ``format`` specifies what format the values in the
 annotations dict should be returned in.
-``format`` accepts the following values, defined as attributes on the
+``format`` accepts the following values, as defined the
 ``inspect`` module::
```

Comment:
```suggestion
``format`` accepts any member of the ``AnnotationFormat`` enumeration,
as defined in the ``inspect`` module::
```

Reply:
Well, if we want to really get down and exacting here, the `format` parameter of `inspect.get_annotations` accepts any value that *compares equal to* any member of the `AnnotationFormat` enumeration as defined in the `inspect` module.  For example, you can pass in `2`, and `get_annotations` will happily return a "forwardref" format dictionary.  So, how might you put that in a readable way?

Reply:
We don't have to explicitly document support for everything that is technically possible. Are there strong use cases for someone using these public inspection methods to _not_ use the enum values?

Reply:
I don't.  I have a strong use case for the compiler not using the enum values, but the compiler isn't a person, and anyway that's an internal implementation detail.

Reply:
Oh, huh.  Now I get UI where I can comment directly.

I've revised this text.  Instead of redefining the enum for some reason, it just says

> The `format` parameter on these two functions accept the same values
> as the `format` parameter on the `__annotate__` magic method
> defined above; however, these `format` parameters also have a default
> value of `inspect.VALUE`.